src/platform-implementation-js/dom-driver/gmail/views/gmail-thread-view.ts

```diff
@@ -625,6 +627,25 @@ class GmailThreadView {
     return view;
   }
 
+  addSubjectButton(button: any) {
```

Comment:
We might use either `ButtonDescriptor` or `Descriptor<ButtonDescriptor>` as the type here.